src/App.tsx

```diff
@@ -1,61 +1,117 @@
+import React, { useState, ChangeEvent } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find((user) => user.id === userId);
+
+  return foundUser || null;
+}
+
+const startingArray: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [sectionValue, setSectionValue] = useState('');
+  const [inputValue, setInputValue] = useState('');
+  const [todoList, setToDoList] = useState<Todo[]>(startingArray);
+
+  const [titleState, setErrorTitleState] = useState(false);
+  const [selectState, setErrorSelectState] = useState(false);
```

Comment:
Boolean variables should have the prefix `is` or `has`. Something like `isTitleError`

Also, what is the `selectState`, `sectionValue` and `inputValue`? Describe your variables better